sdk/servicebus/service-bus/test/perf/track-1/package.json

```diff
@@ -8,6 +8,7 @@
   "license": "ISC",
   "dependencies": {
     "@azure/service-bus": "^1.1.10",
+    "@azure/service-bus-v7": "npm:@azure/service-bus@^7.0.3",
```

Comment:
Wow, did not know you could do this!

Reply:
Learned it yesterday! :)

All hail npm! All hail SOF!!!